src/main/java/org/prebid/server/bidder/grid/GridBidder.java

```diff
@@ -21,6 +25,20 @@ protected Imp modifyImp(Imp imp, ExtImpGrid impExt) {
         if (impExt.getUid() == null || impExt.getUid() == 0) {
             throw new PreBidException("uid is empty");
         }
+
+        final GridExtImp gridExtImp = mapper.mapper().convertValue(imp.getExt(), GridExtImp.class);
```

Comment:
This can potentially rise exception. Pls wrap with `try...catch` to throw `PreBidException`.